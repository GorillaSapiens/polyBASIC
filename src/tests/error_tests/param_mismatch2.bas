DEF a(x,y) 12
b = a()
