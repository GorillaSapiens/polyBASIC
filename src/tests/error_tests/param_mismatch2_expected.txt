SOURCE 2:6, FUNCTION DEF ❮a❯ REQUIRES 2 PARAMS, BUT CALLED WITH 0
